torchtext/csrc/register_bindings.cpp

```diff
@@ -26,9 +26,14 @@ PYBIND11_MODULE(_torchtext, m) {
       .def("forward", &RegexTokenizer::forward);
 
   py::class_<SentencePiece>(m, "SentencePiece")
+      .def(py::init<std::string>())
+      .def("return_content",
```

Comment:
I'd just make this private, since it's specific to to_ivalue only.

Reply:
Done